### DISCUSSION TITLE
GitHub Enterprise Server License for Azure

### DISCUSSION THREAD

**Discussion by rakelton** [community/community#79144] [Enterprise]
Simple question, does anyone know the secret to getting GitHub to respond to requests for an Azure Enterprise Server License? We've created an account and sent the request again thinking that was why it was ignored, still no response.

**Comment by BilalBoudjema** [comment]
# How to Expedite a Response for Azure Enterprise Server License from GitHub

If you're facing delays in getting a response from GitHub regarding an Azure Enterprise Server License, here are some strategies to help you get a quicker response:

## Steps to Take

### 1. Follow-Up Email
- Send a detailed follow-up email to GitHub support or the specific contact for Azure licenses.

### 2. Check Email Filters
- Ensure that responses are not being directed to your spam or junk mail folders.

### 3. Use the Contact Form
- Utilize the official contact forms on GitHub or Azure for your inquiry.

### 4. GitHub Community Forum
- Post your query in the GitHub Community Forum for advice or assistance from the community.

### 5. Social Media Outreach
- Contact GitHub or Azure on their official social media accounts for a potentially faster response.

### 6. Contact Azure Support Directly
- If the issue is Azure-specific, reaching out to Azure support might be more effective.

### 7. GitHub Account Representative
- If you have a dedicated account representative from GitHub, contact them directly.

### 8. Verify Account Details
- Double-check that all the information provided in your request is accurate.

### 9. Network With Peers
- Consult with peers from other organizations who have obtained an Azure Enterprise Server License.

### 10. Patience and Persistence
- Regular, polite follow-ups at reasonable intervals can be effective. Remember that these processes can take time.

## Additional Information

- Response times can vary based on the current volume of requests being handled by GitHub and Azure.
